pillarbox-demo/src/main/java/ch/srgssr/pillarbox/demo/ui/player/PlayerView.kt

```diff
@@ -73,9 +71,17 @@ fun PlayerView(
 ) {
     val playerError by player.playerErrorAsState()
     playerError?.let {
+        val sessionId = remember {
+            if (player is PillarboxExoPlayer) {
+                player.getCurrentPlaybackSessionId() ?: "No session bitch"
+            } else {
+                null
+            }
+        }
         PlayerError(
             modifier = modifier,
             playerError = it,
+            sessionId = sessionId,
```

Comment:
Shouldn't this also be done on the TV app?

Reply:
We could but I guess for TV, it should be better to have a QR code or something else that integrators might use. We already show how to retrieve that SessionId. So in my opinion it is not required here, what do you thing?

Reply:
Fine for me 👍🏻